src/main/java/edu/colorado/plv/fixr/slicing/APISlicer.java

```diff
@@ -221,7 +221,7 @@ public static String getSlicedMethodName(String methodName) {
   private Set<Unit> getSeeds(SlicingCriterion sc)
   {
     Set<Unit> seeds = new HashSet<Unit>();
-    for (Object node : pdg.getNodes()) {
+    for (Object node : pdg){// TODO: test change, used to be: pdg.getNodes()) {
```

Comment:
@ftc what is the difference of iterating directly over the pdg or getting the pdg nodes with pdg.getNodes()?

Reply:
This seems to simply be an API change.  I am going to remove the TODO.